### PR TITLE
Add revised workflow figures

### PR DESCRIPTION
**Pull request by cgreene**
This adds the new workflow figures. I'll also update the legend.

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The process of writing and revising material can be orchestrated through GitHub
 ![
 **Deep Review editing workflow.**
 Any reader can become a contributor by proposing a change through a pull request.
-In this example, the contributor opens an issue to discuss a manuscript modification.
+In this example, where manuscript text is shown in solid lines on the left of the timline and discussion on github is shown by squiggly lines to the right of the timeline, the contributor opens an issue to discuss a manuscript modification.
```
Comment (agitter) [inline content/02.main-text.md]:
```suggestion
In this example, where manuscript text is shown in solid lines on the left of the timeline and discussion on GitHub is shown by squiggly lines to the right of the timeline, the contributor opens an issue to discuss a manuscript modification.
```

**Review by agitter — CHANGES_REQUESTED** [review]
The caption text and figure look good.  Can you also please write a quick response to the reviewer describing why we kept the vertical orientation and what changes you made?  We've been linking the issue and pull request so that we don't have to copy all the text changes into the response letter.

**Comment by cgreene** [comment]
@agitter response to reviewers updated and ready for another look! Thanks :)

**Review by agitter — APPROVED** [review]
Great!

**Pull request merged**